packages/jest-test-sequencer/src/index.ts

```diff
@@ -65,6 +70,23 @@ export default class TestSequencer {
     return cache;
   }
 
+  shard(
+    tests: Array<Test>,
+    options: SortOptions = {shardCount: 1, shardIndex: 1},
+  ): Array<Test> {
+    if (options?.shardCount > 1) {
```

Comment:
options hat default value also is `?` hier unnötig.

warum aber überhaupt default value?